### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi There!

I have made the following changes to the repository: :zap:

- Setup React with Routes. :rocket:
- Setup Linters. :rocket:
- Add the `components` folder. :heavy_plus_sign:
- Add component files: `AddBook.jsx` `Book.jsx` `BookList.jsx` `Categories.jsx` `NavigationHeader.jsx` :heavy_plus_sign:
- Add functionality to the components. :wrench:
- Add routers to `App.jsx` :twisted_rightwards_arrows:

Thank you for taking the time to review my code! :sparkles:

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
```
Comment (mckent05) [inline src/components/Book.jsx]: - Please remove all disabled linters from your codebase, linters help us ensure we write code that is clean and easy to maintain. You can check [here](https://reactjs.org/docs/typechecking-with-proptypes.html) for more information on how to fix this error .Please fix this 🙏

File: src/components/AddBook.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/prop-types */
```
Comment (mckent05) [inline src/components/AddBook.jsx]: - Please remove all disabled linters from your codebase, linters help us ensure we write code that is clean and easy to maintain. You can check [here](https://reactjs.org/docs/typechecking-with-proptypes.html) for more information on how to fix this error .Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @jaimevillegas, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your bookstore components is well setup and renders as expected, good job 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.jsx (hunk 2 above)
Comment (jaimevillegas) [reply src/components/AddBook.jsx]: Thank you so much for the review! I have modified the files that you mentioned, and I added `PropTypes` for each one 🤓👨🏻‍💻🎉

**Review by Tufahel — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Hi @jaimevillegas :wave: 
 
Your project is complete now!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 👏🏻



:heavy_check_mark: Linter checks are passing.
:heavy_check_mark: Readme is professional.
:heavy_check_mark: The app is working fine.
:heavy_check_mark: Commit messages are professional.


## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**